spec/index.bs

```diff
@@ -2379,8 +2438,16 @@ dictionary GPUTextureCopyView {
 };
 </script>
 
+A {{GPUTextureCopyView}} is a view of a sub-region of a [=texture=] [=subresource=] with the initial offset {{GPUOrigin3D}} in
+texels, used when copying data from or to a {{GPUTexture}}.
+
+  - For {{GPUTextureDimension/2d}} textures, data is copied from or to one [=mipmap level=] and one [=array layer=] of the [=texture=].
+  - For {{GPUTextureDimension/3d}} textures, data is copied from or to one [=mipmap level=] of the [=texture=].
```

Comment:
doesn't say anything about a slice of a 3D texture?

Reply:
Oh I haven't implemented copies with 1D or 3D textures in Dawn so I am not sure about the related validation rules. I've left a TODO here. What do you think?

Reply:
Always OK with me to leave things TODO.